### PR TITLE
Feature/coolwallet [v5]

### PR DESCRIPTION
**Pull request by gamalielhere**
### Feature

- [x] Updated CHANGELOG.md
- [x] Add PR label
- [x] No hard-coded strings

### CODE REVIEW

File: src/wallets/hardware/coolwallet/index.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+      if (result) {
+        const resultTx = new Transaction(result);
+        tx.v = getBufferFromHex(sanitizeHex(resultTx.v.toString('hex')));
+        tx.r = getBufferFromHex(sanitizeHex(resultTx.r.toString('hex')));
+        tx.s = getBufferFromHex(sanitizeHex(resultTx.s.toString('hex')));
+        const signedChainId = calculateChainIdFromV(tx.v);
+        if (signedChainId !== networkId)
+          Toast.responseHandler(
+            new Error(
+              'Invalid networkId signature returned. Expected: ' +
```
Comment (gesseekur) [inline src/wallets/hardware/coolwallet/index.js]: heh translation :D

File: src/layouts/AccessWalletLayout/components/HardwareModal/HardwareModal.vue [hunk 2]
```diff
@@ -166,6 +171,15 @@ export default {
...
           link: 'http://lddy.no/a4im'
+        },
+        {
+          name: COOLWALLET_TYPE,
+          imgPath: coolwallet,
+          text: 'CoolWallet',
+          disabled: false,
+          msg: '',
+          link:
+            'https://coolwallet.io/product/coolwallet/?ref=myetherwalletmyetherwallet'
```
Comment (kvhnuke) [inline src/layouts/AccessWalletLayout/components/HardwareModal/HardwareModal.vue]: ref= has myetherwalletmyetherwallet is this right?

File: src/layouts/AccessWalletLayout/components/HardwareModal/HardwareModal.vue [hunk 3]
```diff
@@ -258,6 +272,39 @@ export default {
           this.openXwallet();
           this.$refs.hardware.hide();
           break;
+        case COOLWALLET_TYPE:
+          // eslint-disable-next-line
+          const hasStoredPw = store.get('cwPass') || null;
```
Comment (kvhnuke) [inline src/layouts/AccessWalletLayout/components/HardwareModal/HardwareModal.vue]: why are we saving the pass to localstorage?

File: src/layouts/AccessWalletLayout/components/HardwarePasswordModal/HardwarePasswordModal.vue [hunk 4]
```diff
@@ -78,12 +80,32 @@ export default {
...
-      this.walletConstructor('', this.password)
-        .then(_newWallet => {
-          this.$emit('hardwareWalletOpen', _newWallet);
-        })
-        .catch(this.walletConstructor.errorHandler);
+    async unlockWallet() {
+      if (this.hardwareBrand === 'CoolWallet') {
+        await cwsTransportLib.listen(async (error, device) => {
+          if (device) {
+            const transport = await cwsTransportLib.connect(device);
```
Comment (kvhnuke) [inline src/layouts/AccessWalletLayout/components/HardwarePasswordModal/HardwarePasswordModal.vue]: this should be moved inside the wallet init function? I think previous `unlockWallet` function should be sufficient

File: src/translations/errors-global/en_US.json [hunk 5]
```diff
@@ -8,5 +8,9 @@
   "no-mobile-support": "Hardware wallet not supported by mobile browser!",
   "something-went-wrong": "Something went wrong!",
   "signer-address-different": "Signer address is different from the derived address!",
+<<<<<<< HEAD
```
Comment (kvhnuke) [inline src/translations/errors-global/en_US.json]: left over git stuff

File: src/layouts/AccessWalletLayout/components/HardwarePasswordModal/HardwarePasswordModal.vue (hunk 4 above)
Comment (gamalielhere) [reply src/layouts/AccessWalletLayout/components/HardwarePasswordModal/HardwarePasswordModal.vue]: moving it here fixed the scoping issue

File: src/layouts/AccessWalletLayout/components/HardwareModal/HardwareModal.vue (hunk 2 above)
Comment (gamalielhere) [reply src/layouts/AccessWalletLayout/components/HardwareModal/HardwareModal.vue]: I got this from them so im not too sure. I'll ask

**Pull request merged**